cli.go

```diff
@@ -86,7 +86,8 @@ func CallContext(ctx context.Context, cmd Function, args ...string) int {
 		fmt.Fprintln(Err, err)
 	default:
 		if err != nil {
-			log.Print(err)
+			errorLogger := log.New(Err, "", log.LstdFlags)
```

Comment:
I guess one improvement would be to use a sync.Once at the package level, but 99% of times we're just going to be invoking a single instance of this per process runtime so it doesn't seem worth it.